bigbluebutton-html5/imports/ui/components/connection-status/component.jsx

```diff
@@ -31,7 +33,7 @@ const ConnectionStatus = () => {
     const startTime = performance.now();
     fetch(
       `${getBaseUrl()}/rtt-check`,
-      { signal: AbortSignal.timeout(STATS_INTERVAL) },
+      { signal: AbortSignal.timeout(STATS_TIMEOUT) },
     )
       .then((res) => {
         if (res.ok && res.status === 200) {
```

Comment:
We currently don’t have logs or a `setLastRttRequestSuccess(false)` call when the status isn’t `200`. I believe we should monitor this scenario too.

When a request fails due to a server issue, it returns a `404 Not Found` response rather than triggering the `catch`.